### DISCUSSION TITLE
vendor/psy/psysh/src/ExecutionLoopClosure.php(55) : eval()'d code:1: bool(false)

### DISCUSSION THREAD

**Discussion by punkfairie** [spatie/laravel-permission#2261] [Q&A]
I'm trying to test permission via tinker, and getting this every time.

I have an `Assistant` model that extends `User`, uses the same user table for both. I'm trying to assign permissions to `Assistant` like this:

`{user_id}.{resource}.{action} // e.g. 1.appointments.create`

Sometimes I will need to put a wildcard in `user_id` e.g `*.appointments.create`.

Both `*.appointments.create` and `1.appointments.create` exist, and `1.appointments.create` is assigned to the `Assistant`. I can't check for either permission, both return `vendor/psy/psysh/src/ExecutionLoopClosure.php(55) : eval()'d code:1:
bool(false)`. Google is no help and I can't begin to figure out what this error even means.

**Comment by angeljqv** [comment]
This package is not using `psy/psysh`

**Reply by punkfairie** [reply]
Well the only time I ever get this error is with this package, so clearly something is interacting there.

**Reply by parallels999** [reply]
Make a debug or upload a failing test on a fresh instalation
